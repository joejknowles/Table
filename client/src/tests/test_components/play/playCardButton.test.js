import PlayCardButton from '../../../components/play/playCardButton';
import React from 'react';

import ReactTestUtils from 'react-addons-test-utils';
const shallowRenderer = ReactTestUtils.createRenderer();

it('PlayCardButton renders as before', () => {
  const component = shallowRenderer.render(<PlayCardButton />);
  expect(component).toMatchSnapshot();
});
