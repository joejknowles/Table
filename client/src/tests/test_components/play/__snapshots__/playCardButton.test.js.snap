exports[`test PlayCardButton renders as before 1`] = `
<div
  className="pile">
  <Connect(Component)
    action={
      Object {
        "type": "PLAY_CARD",
      }
    }
    className="play-card playing-card face-down" />
</div>
`;
